packages/ui/v2/core/Dialog.tsx

```diff
@@ -175,12 +141,32 @@ export function DialogHeader(props: DialogHeaderProps) {
 export function DialogFooter(props: { children: ReactNode }) {
   return (
     <div>
-      <div className="mt-5 flex justify-end space-x-2 rtl:space-x-reverse">{props.children}</div>
+      <div className="mt-7 flex justify-end space-x-2 rtl:space-x-reverse ">{props.children}</div>
```

Comment:
We were often wrapping the child of dialog footer with mt-2 so I just merged it here.